AMW_business/src/main/java/ch/puzzle/itc/mobiliar/business/deploy/control/DeploymentNotificationService.java

```diff
@@ -55,44 +52,35 @@ public class DeploymentNotificationService {
 	@Inject
 	private ResourceDependencyResolverService resourceDependencyResolverService;
 
-	@Inject
-	private ReleaseMgmtService releaseMgmtService;
-
 	@Inject
 	private Logger log;
 
 	/**
 	 * Creates a notification email for deployment executions and sends them to
-	 * the given receipients
+	 * the given recipients
 	 *
 	 * @param deployments
 	 *            - the executed deployments
 	 * @return the message to be stored with the deployment.
 	 * @throws AddressException
 	 * @throws MessagingException
 	 */
-	public String createAndSendMailForDeplyoments(
-			final List<DeploymentEntity> deployments) {
+	public String createAndSendMailForDeplyoments(final List<DeploymentEntity> deployments) {
 		String message = null;
-
 		// do nothing if no deployment is available
-		if (deployments != null && !deployments.isEmpty()) {
-			String subjectMessage = "AMW-Deploy for tracking id: "
-					+ deployments.get(0).getTrackingId();
-
-			try {
-				Address[] emailReceipients = getAllReceipients(deployments);
-
-				if (notificationService.createAndSendMail(subjectMessage,
-						getMessageContentForDeployments(deployments),
-						emailReceipients)) {
-					message = getSuccessfulSendMailToReceipientMessage(emailReceipients);
-				}
-			} catch (MessagingException e) {
-				message = getFailureSendMailMessage(e.getMessage());
-				log.log(Level.WARNING, "Deployment notification ("
-						+ subjectMessage + ") could not be sent", e);
+		if (deployments == null || deployments.isEmpty()) {
+			return message;
```

Comment:
So - if deployments is null or empty, null will be returned? is this intentional?

Reply:
I think so. It was returning null before, I just switched the if statement around.